### ISSUE TITLE
[katana/torii] Issues with syncing on torii launch 

### ISSUE THREAD

**Issue by edisontim** [dojoengine/dojo#2574]
When launching torii after katana and after having migrated + setup our world (about 100 txs), torii doesn't sync the last few transactions.
We could fix this temporarily by setting the blocktime of katana to 30s, from 1s where the bug occurs.
To reproduce: 
- https://github.com/BibliothecaDAO/eternum
- copy `season_pass/scripts/deployment/.env.example` to `season_pass/scripts/deployment/.env`
- modify the `./scripts/contracts.sh` the line that runs katana, set the blocktime to 1000 instead of 30000 
- `pnpm i && pnpm run build-packages`
- run from root dir:
  - `sh ./scripts/contracts.sh`
  - `sh ./scripts/indexer.sh --setConfig true`

Look in the torii logs for a store set record for the Bank model, this should be there but isn't present

**Comment by edisontim** [comment]
if the config didn't seem to work: 
- `cd contracts`
- `bun --env-file=../client/.env.local ../config/index.ts `

**Comment by edisontim** [comment]
this is the .env file that should be place in the `client/` dir
```
VITE_PUBLIC_MASTER_ADDRESS="0x127fd5f1fe78a71f8bcd1fec63e3fe2f0486b6ecd5c86a0466c3a21fa5cfcec"
VITE_PUBLIC_MASTER_PRIVATE_KEY="0xc5b2fcab997346f3ea1c00b002ecf6f382c5f9c9659a3894eb783c5320f912"
VITE_PUBLIC_WORLD_ADDRESS="0x320b2713e324fe3125bbc42d85ff69cb3c0908b436fa38a35746dbc45deeb11"
VITE_PUBLIC_ACCOUNT_CLASS_HASH="0x07dc7899aa655b0aae51eadff6d801a58e97dd99cf4666ee59e704249e51adf2"
VITE_NETWORK_FEE_TOKEN="0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7"
VITE_EVENT_KEY="0x1a2f334228cee715f1f0f54053bb6b5eac54fa336e0bc1aacf7516decb0471d"
VITE_PUBLIC_TORII="http://localhost:8080"
VITE_TORII_GRAPHQL="http://localhost:8080/graphql"
VITE_TORII_WS="ws://localhost:8080/graphql/ws"
VITE_PUBLIC_NODE_URL="http://localhost:5050/"
VITE_PUBLIC_DEV=true
VITE_PUBLIC_SHOW_FPS=true
VITE_PUBLIC_GRAPHICS_DEV=false
VITE_PUBLIC_TORII_RELAY="/ip4/127.0.0.1/tcp/9092/ws"










VITE_SEASON_PASS_ADDRESS=0x4d4ded7dd7a8f151189713234fa088a27b3de808a65aaff39c2d61eef63273e
VITE_REALMS_ADDRESS=0x2fc8361a78e1bc73c0bb18a938276b02e3b774669b4c166bedd12fc56f05910
VITE_LORDS_ADDRESS=0x1bbb8f11b85dc29197922c78e80eb24afb0e8bb78868970c0052a7b36bf021e
```

**Comment by kariy** [comment]
should be fixed in #2578 

<img width="933" alt="Screenshot 2024-10-24 at 12 06 02 PM" src="https://github.com/user-attachments/assets/38c2d4cf-5f59-483d-a8b9-d87ff9e2e82e">

**Comment by kariy** [comment]
opening this issue again until it is confirmed by author
